app/src/main/res/layout/recyclerview_row.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+        <TextView
+            android:id="@+id/departureTimeLabel"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:text="@string/departureTimeLabel"
+            style="@style/Text.Subtitle"/>
+        <TextView
+            android:id="@+id/departureTime"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:text="@string/placeholder"
+            style="@style/Text"/>
+        <TextView
+            android:id="@+id/arrivalTimeLabel"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:text="@string/arrivalTimeLabel"
+            style="@style/Text.Subtitle"/>
+        <TextView
+            android:id="@+id/arrivalTime"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
```

Comment:
I'm not sure we want the heights of text views in this file to 'match_parent' perhaps `wrap_content` would make more sense?